### PR TITLE
Mod modified time format

### PR DESCRIPTION
**Pull request by Lilihx**
**Description**
Add setting for exact date format.

Default: previous "from now" format
After setting: display "YYYY-MM-DD HH:mm" format

截屏2021-09-07 下午1 53 14

#1525

### CODE REVIEW

**Comment by o1egl** [comment]
Some people like the current behavior. It must be set as a user config param

File: frontend/src/i18n/en.json [hunk 1]
```diff
@@ -214,6 +215,7 @@
     "rules": "Rules",
     "rulesHelp": "Here you can define a set of allow and disallow rules for this specific user. The blocked files won't show up in the listings and they wont be accessible to the user. We support regex and paths relative to the users scope.\n",
     "scope": "Scope",
+    "setDateFormat": "Set exact date format",
```
Comment (o1egl) [inline frontend/src/i18n/en.json:221]: duplicate of line 30

Comment (Lilihx) [reply frontend/src/i18n/en.json:221]: Actually, I copy the feture "hit dot files" to implement the issue.
there are two "hit dot files" settings in en.json
I'll commit that

File: frontend/src/components/files/ListingItem.vue [hunk 2]
```diff
@@ -106,6 +106,9 @@ export default {
       return filesize(this.size);
     },
     humanTime: function () {
+      if (this.user.dateFormat) {
+        return moment(this.modified).format("YYYY-MM-DD HH:mm");
```
Comment (ramiresviana) [inline frontend/src/components/files/ListingItem.vue]:
```suggestion
        return moment(this.modified).format("L LT");
```
FileBrowser supports multiples languages, dates should be displayed with a localized format.

**Comment by Lilihx** [comment]
great

Ramires Viana ***@***.***> 于2021年9月9日周四 下午8:30写道：

**Pull request merged**